code/src/Pizza.ts

```diff
@@ -0,0 +1,14 @@
+import {Consumable} from "./Consumable";
+
+export class Pizza extends Consumable {
```

Comment:
According to the uml diagram ```Pizza``` class should also have ```slicesEaten``` member, which is incremented inside ```eat``` method.